src/components/Atoms/SearchField/index.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import FormControl from '@mui/material/FormControl'
+import OutlinedInput from '@mui/material/OutlinedInput'
+import Box from '@mui/material/Box'
+import SearchOutlinedIcon from '@mui/icons-material/SearchOutlined'
+import { styled } from '@mui/material'
+
+const CustomSearchField = styled(OutlinedInput)(() => ({
+  height: '25px',
+}))
+
+const CustomizedInputBase: React.FC = () => {
+  return (
```

Comment:
placeholder, input value, onchange handlers can be sent as props

Reply:
added in the new commit